docs/reference/elasticsearch/rest-apis/searching-with-query-rules.md

```diff
@@ -26,6 +26,7 @@ For example, a query rule could be defined to match a user-entered query string
 
 Rules are defined using the [query rules API](https://www.elastic.co/docs/api/doc/elasticsearch/group/endpoint-query_rules) and searched using the [rule retriever](/reference/elasticsearch/rest-apis/retrievers/rule-retriever.md) or the [rule query](/reference/query-languages/query-dsl/query-dsl-rule-query.md).
 
+Alternatively, rules can be managed through the [Query Rules UI](https://www.elastic.co/docs/solutions/search/query-rules-ui).
```

Comment:
```suggestion
Query rules can be created and managed using either:
- [Query rules API]({{es-apis}}v9/group/endpoint-query_rules)
- [Query Rules UI](docs-content://solutions/search/query-rules-ui)

You can search with query rules using either:
- [Rule retriever](/reference/elasticsearch/rest-apis/retrievers/rule-retriever.md)
- [Rule query](/reference/query-languages/query-dsl/query-dsl-rule-query.md)
```

Note the use of:
- variable for API docs URL
- cross-repo syntax for `docs-content` link

Reply:
Because there's 2 options each for managing and querying here, probably easier to read in little lists